### PR TITLE
Counters for broadcasted blob idx and coalesced packets

### PR DESCRIPTION
**Pull request by pgarg66**

### CODE REVIEW

**Comment by pgarg66** [comment]
The intent of these counters is to see if leader's window is increasing rapidly, and if more coalescing can be performed to fine tune it.

File: src/crdt.rs [hunk 1]
```diff
@@ -648,6 +651,7 @@ impl Crdt {
             })
             .collect();
 
+        inc_new_counter!("crdt-broardcast-max_idx", max_idx as usize);
```
Comment (sakridge) [inline src/crdt.rs]: broardcast => broadcast

File: src/crdt.rs [hunk 2]
```diff
@@ -635,6 +637,7 @@ impl Crdt {
                     v.contact_info.tvu,
                     blob.is_coding()
                 );
+                max_idx = cmp::max(max_idx, blob.get_index().unwrap());
```
Comment (sakridge) [inline src/crdt.rs]: the blobs should be in order right? can we just look at the last one?

Comment (sakridge) [reply src/crdt.rs]: or it should just be the same as transmit_index below the errors loop

Comment (pgarg66) [reply src/crdt.rs]: I wan't sure of the in-order thing. If you are certain, I can remove the extra logic.

Comment (pgarg66) [reply src/crdt.rs]: Thanks @sakridge , I think you are right about transmit_index. Changed the code to just use that.

Comment (sakridge) [reply src/crdt.rs]: Yea, pretty sure. Look at the loop that generates the `orders` vec, it iterates from `transmit_index..received_index` and plucks the things out of the window in order.

**Pull request merged**